README.md

```diff
@@ -86,27 +80,126 @@ $POSTGRES_PASSWORD  # postgres's user's password
 ```
 
 If the postgres environment variables are not taking into effect:
+
 ```
 $ docker-compose kill
 $ docker-compose rm
 ```
+
 To access the postgres DB from a workstation:
+
 ```
 $ docker-compose run dev psql -h postgres -U < postgres username from dockerfiles/app.env >
- ```
+```
+
 or from inside the container:
+
 ```
 $ psql -h postgres -U $POSTGRES_USER
 ```
 
+### Running tests
+
+```
+$ docker-compose run --service-ports --rm dev bash
+[root@9eea4caf7740 code]# rake spec
+... tests running ...
+Finished in 0.1669 seconds (files took 0.36978 seconds to load)
+60 examples, 0 failures
+[root@9eea4caf7740 code]# rake
+Migrating to latest
+/usr/local/bin/ruby app.rb
+== Sinatra (v1.4.7) has taken the stage on 4567 for development with backup from Thin
+Thin web server (v1.7.0 codename Dunder Mifflin)
+Maximum connections set to 1024
+Listening on localhost:4567, CTRL+C to stop
+```
+
+You may be prompted to `bundle install`.
+
+## Development Without `docker`
+
+If you do not want to install `docker` and `docker-copmose` and instead want to
+run the service natively you can try installing the following packages:
+
+- `sqlite-devel`
+- `postgresql`
+- `postgresql-devel`
+- `ruby-2.3.0` and `gem`
+- `gem install bundle`
```

Comment:
The `gem install bundle` command shouldn't be listed with the other dependencies